style/build_gecko.rs

```diff
@@ -369,6 +369,7 @@ fn generate_atoms() {
         .join("regen_atoms.py");
     println!("cargo:rerun-if-changed={}", script.display());
     let status = Command::new(&*PYTHON)
+        .env("PYTHONDONTWRITEBYTECODE", "1")
```

Comment:
I don't think this one is necessary because `build_gecko::generate()` isn't run when we can `cargo publish`.

Reply:
Popped that in there for equivalence with the previous code. But yeah, it can probably go.